src/libraries/System.Text.Json/tests/Serialization/ConstructorTests/ConstructorTests.ParameterMatching.cs

```diff
@@ -28,191 +29,191 @@ public ConstructorTests(DeserializationWrapper serializer)
         }
 
         [Fact]
-        public void ReturnNullForNullObjects()
+        public async Task ReturnNullForNullObjects()
         {
-            Assert.Null(Serializer.Deserialize<Point_2D>("null"));
-            Assert.Null(Serializer.Deserialize<Point_3D>("null"));
+            Assert.Null(await Serializer.DeserializeAsync<Point_2D>("null"));
+            Assert.Null(await Serializer.DeserializeAsync<Point_3D>("null"));
         }
 
         [Fact]
-        public void JsonExceptionWhenAssigningNullToStruct()
+        public async Task JsonExceptionWhenAssigningNullToStruct()
         {
-            Assert.Throws<JsonException>(() => Serializer.Deserialize<Point_2D_With_ExtData>("null"));
+            await Assert.ThrowsAsync<JsonException>(() => Serializer.DeserializeAsync<Point_2D_With_ExtData>("null"));
         }
```

Comment:
Remove `async` and return `Assert.ThrowsAsync` instead or perhaps we should add more assertions? The former should not be part of this PR.